src/rnp/rnp.cpp

```diff
@@ -626,6 +626,8 @@ rnp_main(int argc, char **argv)
                 print_usage(usage);
                 ret = EXIT_FAILURE;
                 goto finish;
+            case 'h':
+                [[fallthrough]];
```

Comment:
Please cover this (and below) with some simple CLI test to make codecov happy.

Reply:
Seems that CodeCov is really unhappy by dropping coverage from 80% to 60% 😉